inputPS4.txt

```diff
@@ -0,0 +1,4 @@
+3 5 7 9 11 13 15 17

```

Comment:
Add tests for 
1. negative numbers
2. large numbers
3. odd array length 
4. input with 2 elements with increasing
5. input with 2 elements with decreasing

Reply:
1. it does take into account the negative numbers
2. it does take into account the large numbers
3. changes are added
4. its working already
5. it's working already